java/quarkus/src/main/java/yolo/BenchmarkResource.java

```diff
@@ -1,33 +1,35 @@
 package yolo;
 
+
+import io.smallrye.mutiny.Uni;
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.POST;
 import jakarta.ws.rs.Path;
-import jakarta.ws.rs.PathParam;
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.core.MediaType;
+import org.jboss.logging.Logger;
 
 @Path("/")
 public class BenchmarkResource {
-
+    private static final Logger log = Logger.getLogger(BenchmarkResource.class);
     @GET
     @Path("/")
     @Produces(MediaType.TEXT_PLAIN)
-    public String root() {
-        return "";
+    public Uni<String> root() {
```

Comment:
It would be easier to use the `@NonBlocking` annotation on the method and just return the string